test/Rhetos.LightDMS.IntegrationTest/Utilities/TestDataUtilities.cs

```diff
@@ -126,6 +126,31 @@ public static HttpRequestMessage GenerateUploadRequest(int fileCount = 1)
             blobClient.Upload(fileStream, true);
         }
 
+        public static void SeedS3StorageFile(WebApplicationFactory<Startup> factory,

```

Comment:
Can we also put a file to S3 storage emulator?

Reply:
Updated.